cognee/modules/retrieval/cypher_search_retriever.py

```
@@ -50,7 +52,7 @@ async def get_context(self, query: str) -> Any:
                 logger.warning("Search attempt on an empty knowledge graph")
                 return []
 
-            result = await graph_engine.query(query)
+            result = jsonable_encoder(await graph_engine.query(query))
         except Exception as e:
             logger.error("Failed to execture cypher search retrieval: %s", str(e))
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)